src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0 10px 50px;
+}
+
+.ul__items {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
```

Comment:
You should also set 'align-items: center;' on the header to center the items vertically.